R/graph.R

```diff
@@ -511,12 +511,16 @@ MessageLogger = R6Class(
       if (identical(force, FALSE) && self$isNotLogging()) return(NULL)
       self$reactCache[[reactObj$reactId]] <- reactObj
     },
-    shortenString = function(txt, n = 100) {
+    shortenString = function(txt, n = 250) {
+      if (is.na(txt) || is.null(txt)) {
```

Comment:
You might want to change this to `is.null(txt) && isTRUE(is.na(txt))`, otherwise you'll get a warning if `txt` is actually `NULL`.

Reply:
`&&` or `||` ?

Reply:
Sorry, `||`, heh.